release/pipeline/configs/environment_config.py

```diff
@@ -26,7 +26,7 @@
     MFEST_FILE='build.xml',
     MFEST_COMMIT='master@{{{timestamp}}}',
     VERSION='{major}.{minor}.{patch}-pre{date}-{rc}',
-    GCS_BUILD_BUCKET='istio-release-pipline-data',
+    GCS_BUILD_BUCKET='istio-release-pipeline-data',
```

Comment:
Be sure to create the bucket.

Reply:
Line 46 also has an instance of "pipline", but it's too far away from any other change for github to allow me to comment on it.

Reply:
Yes I have already created the bucket.

Also updated line 46. Failed to take care of the capital P when I grepped.

Reply:
Hrm, the bucket didn't exist when I made the comment.  Anyway, it does seem to exist now.  I'm not sure if the location matters, but for now I'm doing to assume that multi-regional is a strict superset of a specific region (which is what the original used).

Reply:
in which project was the bucket created ?

Reply:
istio-release.  This is also the project used for running the daily builds